WordPress/src/main/res/values/dimens.xml

```diff
@@ -128,4 +129,19 @@
     <dimen name="stats_button_corner_radius">4dp</dimen>
     <dimen name="stats_barchart_height">128dp</dimen>
 
+    <!-- my site -->
+    <dimen name="my_site_add_button_padding_bottom">5dp</dimen>
```

Comment:
This seems like a lot of dimensions to add for one screen. Can they be consolidated? For example, there are four dimens that are `10dp`. Also, are these values coming from the mockups? If not, can we use the pre-existing 'magin_xxx` values instead?